src/modules/user-conditions/UserConditions.module.scss

```diff
@@ -70,12 +73,13 @@ $terms-of-service-background-color: #ebfbf8;
 
 	&__content {
 		margin-top: $spacer-1xl;
-		max-height: 100%;
 		min-height: 0;
 		overflow-y: auto;
+		max-height: calc(100vh - $top-navigation-bar-height - $header-and-buttons-height-mobile);
 
 		@include respond-at("md") {
 			margin-top: $spacer-lg * 2;
+			max-height: calc(100vh - $top-navigation-bar-height - $header-and-buttons-height);
```

Comment:
Not a fan of big `calc()`'s like this, no alternatives here?

Reply:
yeah i agree
i'm going to look into grid fill height or something similar when i work on the detail page